### ISSUE TITLE
add formatting pre-commit check

### ISSUE THREAD

**Issue by LakshyaSatpal** [PalisadoesFoundation/talawa-api#1778]
**Is your feature request related to a problem? Please describe.**
We don't check correct prettier formatting of code before commit, though we do have lint checks.

**Describe the solution you'd like**
Use the scripts defined in package.json and check for formatting before every commit.

**Potential internship candidates**
Please read this if you are planning to apply for a Palisadoes Foundation internship https://github.com/PalisadoesFoundation/talawa/issues/359

**Comment by palisadoes** [comment]
Closing. This doesn't seem valid

**Comment by palisadoes** [comment]
1. Sorry about the hasty closing of the issue and PR
2. Your PR didn't address the formatting. Why?

**Comment by LakshyaSatpal** [comment]
1. on the first commit in the PR, I added `npm run format:fix` in the husky file, which fixes formatting in all the code files.
2. but then I found that lint staged is already configured to format the **staged** files. And, it works fine. So, I restored the husky file.
![image](https://github.com/PalisadoesFoundation/talawa-api/assets/81241551/6520b2f7-34b2-4a71-9d57-22dc084ef587)

So, if we want to format **all code** before committing, issue remains open. Else, we can close this.

(In talawa-admin, pre-commit formats full code https://github.com/PalisadoesFoundation/talawa-admin/blob/main/.husky/pre-commit )

PS: The issue was raised because I'd to fix the formatting and re-push my changes in my other PR #1774, maybe I was not on the latest code, and that's why the GitHub PR workflow format check was failing.
